app/src/org/commcare/views/widgets/AbstractUniversalDateWidget.java

```diff
@@ -436,7 +450,7 @@ private void updateDateDisplay(long millisFromJavaEpoch) {
     /**
      * Update the widget helper date text (useful for those who don't know the other calendar)
      */
-    private void updateGregorianDateHelperDisplay() {
+    protected void updateGregorianDateHelperDisplay() {
```

Comment:
Since you've exposed this method to subclasses, I would rename it to not include the word `Helper`